columnphysics/icepack_fsd.F90

```diff
@@ -564,7 +593,6 @@ subroutine fsd_lateral_growth (dt,        aice,         &
 
       ! Use remaining ice volume as in standard model,
       ! but ice cannot grow into the area that has grown laterally
```

Comment:
Should the comment "but ice cannot grow into the area that has grown laterally" be deleted, since the line below it was?